app/services/git_based_domain_import_service.rb

```diff
@@ -62,6 +81,16 @@ def import(git_repo_id, branch_or_tag, tenant_id)
     domain.update_attribute(:enabled, true)
   end
 
+  def refresh(git_repo_id)
```

Comment:
@gmcculloug I think the repository id is needed for the Queue, we don't need the whole object. I wanted to stay consistent with the other methods in the same file